libs/ui/src/components/atoms/input/style.ts

```diff
@@ -0,0 +1,73 @@
+import styled from 'styled-components';
+
+import { Theme } from '../../../types/theming';
```

Comment:
I ended up resolving this similarly locally -- we can sync about this later because we should be able to streamline the import a bit more